.all-contributorsrc

```diff
@@ -874,6 +874,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "bsieber-mozilla",
+      "name": "bsieber-mozilla",
+      "avatar_url": "https://avatars.githubusercontent.com/u/53873986?v=4",
+      "profile": "https://github.com/bsieber-mozilla",
+      "contributions": [
+        "code"
```

Comment:
I think you should add `"staff"` as well and regen.

Reply:
I had added "code" and "staff" per https://github.com/taskcluster/taskcluster/issues/4955#issuecomment-912104824

Let me close try again.🤞

Reply:
The docs mention the bot, but based on the [changes here](README.md); I'm thinking that others were not using the bot.